distribution/client/test/storage.test.ts

```diff
@@ -56,11 +56,17 @@ describe('The storage module', () => {
       const pluginPath = Storage.pluginsDirectory();
```

Comment:
Pretty sure this is because of the `forEach`. Which could mean that nothing was really tested already here :fearful:.

I just added `expect.assertions(filenames.length*3);`  after this line, and I get 

```jest
  ● The storage module › removePlugins() › should remove all files in a list

    expect.assertions(12)

    Expected twelve assertions to be called but received zero assertion calls.

      54 |     it('should remove all files in a list', async () => {
      55 |       const filenames = ['first', 'second', 'third', 'fourth'];
    > 56 |       expect.assertions(filenames.length*3);
         |              ^
      57 |       const pluginPath = Storage.pluginsDirectory();
      58 |       mkdirp.sync(pluginPath);
      59 |       filenames.forEach((filename) => {

      at Object.it (test/storage.test.ts:56:14)
```

Reply:
I believe reviewing and merging https://github.com/jenkins-infra/evergreen/pull/325 first should help.